packages/aila/src/protocol/schemas/quiz/quizV2.ts

```diff
@@ -82,3 +80,35 @@ export type QuizV2QuestionShortAnswer = z.infer<
 >;
 export type QuizV2QuestionMatch = z.infer<typeof QuizV2QuestionMatchSchema>;
 export type QuizV2QuestionOrder = z.infer<typeof QuizV2QuestionOrderSchema>;
+
+// ********** LLM-SPECIFIC SCHEMAS (MULTIPLE CHOICE ONLY) **********
+
+// LLM can only generate multiple choice questions, so we create specific schemas for that
+export const QuizV2MultipleChoiceOnlyQuestionSchema =
+  QuizV2QuestionMultipleChoiceSchema;
+
+export const QuizV2MultipleChoiceOnlySchema = z.object({
+  version: z.literal("v2").describe("Schema version identifier"),
+  questions: z
+    .array(QuizV2MultipleChoiceOnlyQuestionSchema)
+    .describe("Array of multiple choice quiz questions"),
```

Comment:
Is it helpful to have min/max in description here? Or in the 'WithoutLength' schema below? I guess we want to minimise changes to only necessary so might not want to add in this PR